__tests__/app.test.js

```diff
@@ -33,4 +33,63 @@ describe('/api/topics', () => {
             })
         })
     })
+})
+
+describe('/api', () => {
+    test('GET: 200 sends an object describing all available endpoints to the user', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({ body }) => {
+            for (let endpoint in body) {
+                expect(body[endpoint]).toMatchObject({
+                    description: expect.any(String),
+                    queries: expect.any(Object),
+                    requestFormat: expect.any(Object),
+                    exampleResponse: expect.any(Object)
+                })
+            }
+        })
+    })
+})
+
+describe('/api/articles', () => {
+    describe('api/articles/:article_id', () => {
+        test('GET:200 sends an article object to the user with the correct article_id', () => {
+            return request(app)
+            .get('/api/articles/2')
```

Comment:
You are specifying id 2, you specifically want article 2. Does your test give you any confidence that you have article 2 and its corresponding information?